client/src/components/JurisdictionAdmin/BatchRoundSteps/BatchRoundTallyEntry/BatchDetail.tsx

```diff
@@ -494,59 +507,66 @@ const BatchResultTallySheet: React.FC<IBatchResultTallySheetProps> = ({
         aria-labelledby={`${Classes.TAB_PANEL}_${batch.name}_${selectedTabId}`}
         role="tabpanel"
       >
-        <BatchResultTallySheetTable>
-          <thead>
-            <tr>
-              <th>Choice</th>
-              <th>Votes</th>
-            </tr>
-          </thead>
-          <tbody>
-            {contest.choices.map((choice, i) => (
-              <tr key={choice.id}>
-                <td>{choice.name}</td>
-                <td>
-                  {isTotalsSheet ? (
-                    <span>
-                      {sum(
-                        sheets.map(sheet => sheet.results[choice.id] ?? 0)
-                      ).toLocaleString()}
-                    </span>
-                  ) : isEditing ? (
-                    <input
-                      aria-label={`${choice.name} Votes`}
-                      // Should be fine accessibility-wise, having read and considered the
-                      // accessibility warning under
-                      // https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#autofocus,
-                      // since we're auto-focusing after a relevant user action and the input is
-                      // clearly labeled
-                      // eslint-disable-next-line jsx-a11y/no-autofocus
-                      autoFocus={!isSelectedSheetNewAndUnsaved && i === 0}
-                      className={classnames(
-                        Classes.INPUT,
-                        errors.results?.[choice.id] && Classes.INTENT_DANGER
-                      )}
-                      name={`results[${choice.id}]`}
-                      readOnly={isSubmitting}
-                      ref={register({
-                        min: 0,
-                        required: true,
-                        valueAsNumber: true,
-                      })}
-                      type="number"
-                    />
-                  ) : (
-                    <span>
-                      {(
-                        selectedSheet?.results[choice.id] ?? ''
-                      ).toLocaleString()}
-                    </span>
-                  )}
-                </td>
+        {contests.map(contest => (
+          <BatchResultTallySheetTable key={contest.id}>
```

Comment:
Because choice IDs are unique across contests and tallies are submitted keyed by choice ID alone, e.g.,

```ts
{
  name: 'Sheet 1',
  results: {
    'choice-id-1': 1,
    'choice-id-2': 2,
    'choice-id-3': 3,
    'choice-id-4': 4,
  },
}
```

form submission logic didn't require any updates :tada:

Choice IDs are guaranteed to be unique across contests because `id` is the primary key on the `ContestChoice` model:

https://github.com/votingworks/arlo/blob/237ed24995f5dbb659e1d16f9da1b06911a2deae/server/models.py#L491

Reply:
Feels like it would be maybe a bit clearer if we separated out the results by contest (both here and in the batch tally sheets data model). But it's also nice that this change is pretty minimal. Have you thought about this tradeoff at all?

Reply:
Sounds good! I figured it was on your radar